SA3/qa_web/templates/qa_web/answerspage.html

```diff
@@ -40,16 +40,39 @@
     </style>
   </head>
   <body>
-  <h1>{{ currentQuestion.title }}</h1>
-  <p>{{ currentQuestion.content }}</p>
-  <h2>Question by {{ currentQuestion.owner }}</h2>
-    <div id="wmd-button-bar"></div>
-    <textarea id="wmd-input" class="wmd-input"></textarea>
-    <div id="wmd-preview" class="wmd-panel wmd-preview"></div>
-  <script>
-      var converter = Markdown.getSanitizingConverter();
-      var editor = new Markdown.Editor(converter);
-      editor.run();
-    </script>
+  <!-- Display the question's data -->
+  <div id="question">
+      <h1>{{ currentQuestion.title }}</h1>
+      <p>{{ currentQuestion.content }}</p>
+      <h2>Question by {{ currentQuestion.owner }} on {{ currentQuestion.creation_date }}</h2>
+  </div>
+  <div id="answers">
+  <!-- Display the answers -->
+    {% for answer in answers %}
+        <div id="answer_{{ answer.id }}">
+            <p>{{ answer.content }}</p>
+            <h3>Answer by {{ answer.owner }} on {{ answer.creation_date }}</h3>
+        </div>
+    {% empty %}
+      <h3>Be the first to answer this question!</h3>
+    {% endfor %}
+  </div>
+  <!-- Display the answering form for logged in users -->
+  {% if user.is_authenticated %}
+    <form id="answer_form" method="post">
+        {% csrf_token %}
+        <div id="wmd-button-bar"></div>
+        <textarea id="wmd-input" name="content" class="wmd-input"></textarea>
+        <div id="wmd-preview" class="wmd-panel wmd-preview"></div>
+        <input type="submit" value="Submit your answer"/>
+        <script>
+            var converter = Markdown.getSanitizingConverter();
+            var editor = new Markdown.Editor(converter);
+            editor.run();
+        </script>
```

Comment:
It seems like we do not have the necessary `div` to support markdown once an answer is submitted. This is a front end problem and should be examined as part of enhancement in Sprint 3. Maybe #43.

Reply:
See picture below for visual.